contributors.md

```diff
@@ -91,12 +91,11 @@ The following people are not part of the project team, but have been contributin
 * (marcovmun)
 * Sven Slootweg (joepie91)
 * Daniel Trujillo Viedma (gDanix)
+* Niels NTG Poldervaart (Niels-NTG) - Screenshot filenames
```

Comment:
This is the only one with a description, should it stay?

Reply:
Yes. The problem is that the other entries lacks descriptions, not that this entry has one.